packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/test.ts

```diff
@@ -60,34 +58,37 @@ export function getTestConfig(wco: WebpackConfigOptions<WebpackTestOptions>) {
     resolve: {
       mainFields: [
         ...(wco.supportES2015 ? ['es2015'] : []),
-        'browser', 'module', 'main'
-      ]
+        'browser', 'module', 'main',
+      ],
     },
     devtool: buildOptions.sourceMap ? 'inline-source-map' : 'eval',
     entry: {
-      main: path.resolve(root, buildOptions.main)
+      main: path.resolve(root, buildOptions.main),
     },
     module: {
-      rules: [].concat(extraRules as any)
+      rules: extraRules,
     },
     plugins: extraPlugins,
     optimization: {
-      // runtimeChunk: 'single',
       splitChunks: {
-        chunks: buildOptions.commonChunk ? 'all' : 'initial',
+        chunks: ((chunk: { name: string }) => chunk.name !== 'polyfills'),
         cacheGroups: {
           vendors: false,
           vendor: {
             name: 'vendor',
             chunks: 'initial',
-            test: (module: any, chunks: Array<{ name: string }>) => {
-              const moduleName = module.nameForCondition ? module.nameForCondition() : '';
-              return /[\\/]node_modules[\\/]/.test(moduleName)
-                && !chunks.some(({ name }) => name === 'polyfills');
+            test: (module: { nameForCondition?: () => string }) => {
```

Comment:
This should be able to be reduced to `test: /[\\/]node_modules[\\/]/`

Reply:
Coolio, done.